dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -2501,10 +2501,10 @@ $width: 400px;
   position: fixed;
   top: 0;
   right: 0;
-  color: white;
+  color: $white;
   background-color: $light_purple;
   padding: 3px;
   z-index: 100;
   cursor: pointer;
-  box-shadow: 0 0 10px 0 rgba(0, 0, 0, 0.75);
+  box-shadow: 0 0 7px 2px $shadow;
```

Comment:
Yup!  This is just the environment tag:

![screen shot 2016-08-09 at 10 13 49 am](https://cloud.githubusercontent.com/assets/413693/17525967/fdcb8dd0-5e19-11e6-8f2c-d52d5b7d4309.png)
I tweaked the metrics a little bit in order to re-use the existing shadow color.